jina/hubble/progress_bar.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import time
+from typing import Optional
+from ..logging.profile import TimeContext
+from ..helper import colored, get_readable_size, get_readable_time
+
+
+class ProgressBar(TimeContext):
```

Comment:
why creating a progressbar again? we have one already in `jina/logging/profile.py`